docs/package_integration/uds-package-practices.md

```diff
@@ -1,15 +1,26 @@
-# UDS Package Practices
+# UDS Package Integration Guide
```

Comment:
would prefer to have one "guide" to reduce confusion on naming (also would argue this docs  was meant more to be a summary of requirements than a guide on how to integrate with UDS - that was more what I meant than specific wording on that line)